src/interface_r/R/model.R

```diff
@@ -12,13 +12,21 @@ attach_attrs_to_model <- function(r_model_obj) {
   attrs_exclude_from_attach <- c(
     "fit", "fit_predict", "fit_transform", "score", "predict", "transform",
     "init")
-  model_attrs <- names(r_model_obj$model$model)
+  if(grepl("H2O", as.character(r_model_obj$model))){
```

Comment:
Nitpick: Space missing after `if(`.  There are probably more code formatting issues scattered throughout the code, so we can ignore for now if you want and apply [lintr](https://github.com/jimhester/lintr) at some point before release to clean up spacing.

Reply:
I think a lintr is the right way to go. I can do that in a sep PR